scheduler/src/cook/caches.clj

```diff
@@ -44,4 +44,9 @@
   (-> (CacheBuilder/newBuilder)
     (.maximumSize (get-in config/config [:settings :passport :job-uuid-cache-set-size]))
     (.expireAfterAccess (get-in config/config [:settings :passport :job-uuid-cache-expiry-time-hours]) TimeUnit/HOURS)
-    (.build)))
\ No newline at end of file
+    (.build)))
+(mount/defstate ^Cache job-uuid->job-map :start
+                (-> (CacheBuilder/newBuilder)
+                  (.maximumSize (get-in config/config [:settings :passport :job-uuid-cache-set-size]))
+                  (.expireAfterAccess (get-in config/config [:settings :passport :job-uuid-cache-expiry-time-hours]) TimeUnit/HOURS)
+                  (.build)))
```

Comment:
Yeah-- I sort of took a half-measure here and wanted to discuss it more. The reason I wrote the cache building like this for now is that I wasn't sure if we would want in the future for the `instance-uuid->job-uuid` and `job-uuid->job-map` caches to have different expiry times and sizes. So, I built each one separately but using the same parameters, so that if we did want different expiry and sizes we could easily sub them in.

However, after thinking about it I think that we will always want the two caches to have the same expiry time and size b/c they are currently always used in tandem. If that line of thinking makes sense (I might just be overthinking this) I'll make a helper function to build them.

Reply:
I think that line of thinking makes sense 👍🏼